src/main/resources/GoogleApps/GoogleAppsConfigClass.js

```diff
@@ -1,119 +1,119 @@
+			reactivate(cookieInputs);
+		} else {
+			deactivate(cookieInputs);
+		}
+	}
+
+	function updateDomainHint() {
+		if (typeof(this.value)=='undefined') return;
+		var domainHint = jQuery('#googleapps-domain-livehint');
+		var valid = /^[a-zA-Z0-9][a-zA-Z0-9-]{1,61}[a-zA-Z0-9](?:\.[a-zA-Z]{2,})+$/.test(this.value);
+		if (this.value.length > 0) {
+			if (valid) {
+				domainHint.text(hintTextOn.replace('\{0\}', this.value));
+				domainHint.removeClass('warningmessage');
+			} else {
+				domainHint.text(hintTextInvaliddomain.replace('\{0\}', this.value));
+				domainHint.addClass('warningmessage');
+			}
+		} else {
+			domainHint.text(hintTextOff);
+			domainHint.removeClass('warningmessage');
+		}
+	}
+
+
+	var allInput, cookieInputs;
+	var hintTextOn = "${escapetool.javascript($services.localization.render('GoogleApps.GoogleAppsConfigClass_domain.hintTextOn'))}",
+			hintTextOff = "${escapetool.javascript($services.localization.render('GoogleApps.GoogleAppsConfigClass_domain.hintTextOff'))}",
+			hintTextInvaliddomain = "${escapetool.javascript($services.localization.render('GoogleApps.GoogleAppsConfigClass_domain.hintTextInvaliddomain'))}";
+
+	function updateScopes() {
+		var scopes = "";
+		jQuery(("input[name^='scope_']")).each(function() {
+			if(jQuery.attr(this, "disabled")!=="disabled" && this.checked)
+				scopes = scopes + " " + this.name.substring('scope_'.length);
+		});
+		jQuery("input[name$='_scope']").val(scopes);
+		updateDriveNowWhat();
+	}
+
+	function readScopes() {
+			var scope = jQuery("input[name$='_scope']").val();
+			jQuery(("input[name^='scope_']")).each(function() {
+				if(jQuery.attr(this, "disabled")!=="disabled") {
+					var n = this.name.substring('scope_'.length);
+					if (scope.indexOf(n) > -1) {
+						this.checked = true;
+					} else {
+						this.checked = false;
+					}
+				}
+			});
+			updateDriveNowWhat();
+	}
+
+	function updateDriveNowWhat() {
+			var driveInput = jQuery("input[name='scope_drive']")[0];
+			if(driveInput.checked) jQuery("#driveOnNowWhat").show();
+			else jQuery("#driveOnNowWhat").hide();
+	}
+
+	(function () {
+		// register listeners
+		cookieInputs = jQuery(prefix + 'skipLoginPage, ' + prefix + 'authWithCookies, ' + prefix + 'cookiesTTL');
+		jQuery(prefix + 'useCookies').each(updateCookieFields).change(updateCookieFields);
+
+		var keepThem = "name$=\'_activate\'],[name=\'formactionsac\'],[name=\'form_token\'],"
+			+ "[name=\'xcontinue\'],[name=\'xredirect\'";
+
+		allInputs = jQuery(
+			'#googleapps_GoogleApps\\.GoogleAppsConfig input:not(['+keepThem+'])');
+		jQuery(prefix + 'activate').each(updateAllInputs).change(updateAllInputs);
+
+		jQuery(prefix + 'domain').each(updateDomainHint).on('change keyup', updateDomainHint);
+
+		jQuery(("input[name^='scope_']")).each(function() {
+			jQuery(this).change(function() { updateScopes(); });
+		});
+		readScopes();
+
+		jQuery('#googleapps_GoogleApps\.GoogleAppsConfig').submit(cancelCancelEdit);
+	}).defer();
```

Comment:
This looks really bad.. @polx you shouldn't use tabs for indenting. You probably did this from an IDE so you need to configure your IDE to use 2 plain spaces for JavaScript code. @acotiuga didn't you notice this?

Reply:
@mflorea I used the `w=1` to ignore white spaces, as I usually do when reviewing code. Not a good practice when checking indentation commits.

Reply:
Well... That's a fun one.
The indenting is actually correct: But it is made of tabs which shows correctly if you open in an editor that shows tabs as two spaces as one should when viewing using these conventions. There's two tabs behind jQuery, as it should, for example. GitHub seems to assume 4 spaces per tabs when displaying and then it shows bad. Selecting actually shows it right (one tab). 


Now, I should have detabbed before committing and this would not have happened. I agree.
I've now committed these two files with detabbing (4a62be97a4a72ce6c909f76b4a2b48ba8c085786).

Reply:
Thanks @polx